docs/deployment/config_descriptions.md

```diff
@@ -4,51 +4,106 @@ title: Configuration Descriptions
+```yaml
+# Config for 'dynamos.\<name\>.read' where `name = UsersTable`
+dynamos:
+  UsersTable:
+    read: 5
+```
 
-## useNgapPermissionBoundary
+### config.yml Explained
+
+| field | default     | description
+| ----- | ----------- | -----------
+| prefix | (required) | the name used as a prefix in all aws resources
+| prefixNoDash | (required) | prefix with no dash
+| users | | List of URS usernames permitted to access the Cumulus dashboard/API
+| urs_url | `https://uat.urs.earthdata.nasa.gov/` | URS url used for OAuth
+| useNgapPermissionBoundary | false | Required to be `true` when deploying to the NGAP platform
+| useWorkflowLambdaVersions | true | Version deployed lambdas when they are updated.
+| cmr.username | (required) | the username used for posting metadata to CMR
+| cmr.provider | CUMULUS | the provider used for posting metadata to CMR
+| cmr.clientId | CUMULUS | the clientId used to authenticate with the CMR
+| cmr.password | (required) | the password used to authenticate with the CMR
+| buckets | (required) | Configuration of buckets with key, bucket name, and type (i.e. internal, public private)
+| system_bucket | `buckets.internal.name` | the bucket used for storing deployment artifacts
+| shared_data_bucket | cumulus-data-shared | bucket containing shared data artifacts e.g. Cumulus message adapter
+| ems.provider | CUMULUS | the provider used for sending reports to EMS
+| vpc.vpcId | (required if ecs is used) | the vpcId used with the deployment
+| vpc.subnets | (required) | the subnets used
+| vpc.cidrIp | (required) | the IPv4 CIDR of the VPC
+| ecs.amiid | ami-9eb4b1e5 | amiid of an optimized ecs instance (differnet for each region)
+| ecs.instanceType | (required) | the instance type of the ec2 machine used for running ecs tasks
+| ecs.volumeSize | 50 | the storage on ec2 instance running the ecs tasks
+| ecs.availabilityZone | us-east-1a | the availibity zone used for launching ec2 machines
+| ecs.minInstances | 1 | min number of ec2 instances to launch in an autoscaling group
+| ecs.desiredInstances | 1 | desired number of ec2 instances needed in an autoscaling group
+| ecs.maxInstances | 2 | max number of ec2 instances to launch in an autoscaling group
+| es.name | es5 | name of the elasticsearch cluster
+| es.elasticSearchMapping | 4 | version number of the elasticsearch mapping used
+| es.version | 5.3 | elasticsearch software version
+| es.instanceCount | 1 | number of elasticsearch nodes
+| es.instanceType | t2.small.elasticsearch | size of the ec2 instance used for the elasticsearch
+| es.volumeSize | 35 | the storage used in each elasticsearch node
+| sns.\<name\> | | name of the sns topic
+| sns.\<name\>.subscriptions.\<subscription_name\>.endpoint | | lambda function triggered for each message in the topic
```

Comment:
Can we add the same message here about `(see `@cumulus/deployment/app/config.yml` for examples of core usage)`?